pages/common/ping.md

```diff
@@ -21,3 +21,7 @@
 - Ping host and ring the bell when a packet is received (if your terminal supports it):
 
 `ping -a {{host}}`
+
+- Also display a message if no response was recieved:
+
+`ping -O {{host}}`
```

Comment:
Just interested, what OS is this? 🤔 I can't seem to find it in man pages.

Reply:
What OS? I'm using Ubuntu 18.04 Bionic Beaver. Don't know if this works in other OSes, but i'm pretty sure it does - except windows, of course - the flags are completely differerent there.

Reply:
Hmm. Seems to work fine on my Ubuntu server.

But not on some other Linux versions. Anyway, I think this is pretty useful. :+1:

Reply:
`ping -V` yields `ping utility, iputils-s20161105`.

Reply:
Hrm, that's odd. Maybe it's a Debian/GNU-specific thing or something?